File: src/main/java/com/love/guess/Game.java
package com.love.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random=new Random();
        int secret=random.nextInt(10)+1;
        System.out.println(secret);
        Scanner scanner=new Scanner(System.in);
        int number=0;
        while (number!=secret) {
            System.out.print("Please enter a number: ");
            number=scanner.nextInt();
            if(number<secret){
                System.out.println("higher");
            }else if(number>secret){
                System.out.println("lower");
            }else {
                System.out.println("Great,the number is "+number);
            }
        }
    }
}

File: src/main/kotlin/com/love/Game.kt
package com.love

import java.util.*

fun main(args: Array<String>) {
    val secret=Random().nextInt(10)+1
    println(secret)
    val scanner=Scanner(System.`in`)
    var number=0
    while (number!=secret) {
        print("Please enter a number: ")
         number=scanner.nextInt()
        if(number<secret) println("higher")
        else if(number>secret) println("lower")
        else println("Great,the number is $number")
    }
}